frontend/src/visualizations/Scalar.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component, PropTypes } from "react";
+
+import { formatScalar } from "metabase/lib/formatting";
+
+export default class Scalar extends Component {
+    static displayName = "Number";
+    static identifier = "scalar";
+    static iconName = "number";
+
+    static isSensible(cols, rows) {
+        return rows.length === 1 && cols.length === 1;
+    }
+
+    static checkRenderable(cols, rows) {
```

Comment:
`checkRenderable` throws an error if it this viz can't be rendered for this data, and scalars didn't need  a check, so it's blank. I can add a comment.

This API isn't set in stone, BTW. I just wanted to get the basic idea working.